examples/ssx-quickstart-express/src/index.ts

```diff
@@ -0,0 +1,25 @@
+import express from 'express';
+import http from 'http';
+import { AddressInfo } from 'net';
+import { SSXServer, SSXExpressMiddleware } from '@spruceid/ssx-server';
+import cors from 'cors';
+
+const app = express();
+
+const ssx = new SSXServer({
+  signingKey: "SUPER SECRET",
```

Comment:
I would say to add dotenv as a dependency and to use it to import the secret. So someone copying this repo has this security practice and doesn't commit secrets
```suggestion
  signingKey: process.env.SIGNING_KEY || "SUPER SECRET",
```

Reply:
Mmmm, since this is supposed to come from the guide the idea was to keep it as simple as possible